### PR TITLE
Add PPID field to `execsnoop.bt`

### PR DESCRIPTION
**Pull request by basil**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,8 @@ and this project adheres to
 ## Unreleased
 
 #### Added
+- Add PPID field to `execsnoop.bt`
```
Comment (jordalgo) [inline CHANGELOG.md:12]: This should go in the "tools" subsection at the bottom.

**Review by jordalgo — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (basil) [reply CHANGELOG.md]: commit 1ca2529b

**Comment by basil** [comment]
commit 58aa5204

**Review by jordalgo — APPROVED** [review]

**Review by danobi — APPROVED** [review]
thanks!

**Comment by danobi** [comment]
Oh, I think `tools/execsnoop_example.txt` needs to be updated

**Comment by basil** [comment]
commit f35f21a9

**Pull request merged**